feat: app settings (#1994)
appsmithorg/appsmith-docs: website/docs/core-concepts/building-ui/designing-an-application/app-theming.md

  </iframe>
</div>

### Theme properties

Determines the overall look and feel of your app, and these settings offer flexibility in tailoring the theme to your preferences. Each app comes with an applied theme that serves as the default visual style. You have two primary ways to modify this applied theme:

**Changing the Theme:** You can switch to a different theme from the saved list of themes or choose from default themes. This quick and easy method allows you to experiment with various visual styles effortlessly.

**Customizing the Theme:** For a more tailored approach, the theme can be further customized based on the available options in the theme properties. These customization options provide granular control over aspects such as fonts, colors, borders, and shadows.

### Font

#### App font

Allows you to modify the fonts universally across the entire application. You can select a font from the list to apply it across all elements in the app.

### Color

#### Primary color

Sets the main color for all components, like buttons, creating a consistent theme throughout the app.

#### Background color

> Allows you to set the page's color / app's background (canvas). This customization option provides control over the backdrop, allowing you to create a specific ambiance or align with brand colors. 

Reviewer: can we rephrase this to be search friendly?

```suggestion
Allows you to set the background colour of the page or application. This customization option provides control over the backdrop, allowing you to create a specific theme or align it with brand colors. 
```
